### PR TITLE
PMM-5702 Introduce STT

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
First mentions of STT (Security Threat Tool) in context of PMM architecture.

### CODE REVIEW

File: source/.res/replace.txt [hunk 1]
```diff
@@ -94,6 +94,7 @@
 .. |pmm-linux-metrics.service| replace:: :file:`pmm-linux-metrics-42000.service`
 .. |pmm-mysql-metrics.service| replace:: :file:`pmm-mysql-metrics-42002.service`
 .. |pmm-mysql-queries.service| replace:: :file:`pmm-mysql-queries-0.service`
+.. |pmm-platform| replace:: PMM Platform
```
Comment (alexanderblinkov-percona) [inline source/.res/replace.txt:97]: I wonder if there should be Percona Platform but not PMM Platform - @rnovikovP please note if it should be changed

Comment (alexanderblinkov-percona) [reply source/.res/replace.txt:97]: And the same point for all the PR

Comment (rnovikovP) [reply source/.res/replace.txt:97]: yeah, this better to be Percona Platform   +  it's more communicating with PMM Server, so let's now move it to pmm server side from pmm client.  More clear documentation is to come.

File: source/manage/client-proxysql-metrics.rst [hunk 2]
```diff
@@ -34,14 +34,14 @@ The output of this command may look as follows:
...
-priority. Here is the previous example modified to use these flags for both port and socket connections::
+service address with the following flags: ``--service-name``, and ``--host`` (the
+hostname or IP address of the service) and ``--port`` (the port number of the
+service), or ``--socket`` (the UNIX socket path). If both flag and positional argument are present, flag gains higher
+priority. Here is the previous example modified to use these flags for both host/port or socket connections::
 
      pmm-admin add proxysql --username=pmm --password=pmm --service-name=my-new-proxysql --host=127.0.0.1 --port=3306 
 
-     pmm-admin add proxysql --username=pmm --password=pmm --service-name=my-new-proxysql --host=127.0.0.1 --socket=/var/run/unix_socket
+     pmm-admin add proxysql --username=pmm --password=pmm --service-name=my-new-proxysql --socket=/var/run/unix_socket
```
Comment (rnovikovP) [inline source/manage/client-proxysql-metrics.rst:44]: Socket option for  ProxySQL not yet   Tested and merged  so it'a better to keep it   until it's done on DEV  and QA side

Comment (PaulJacobs-percona) [reply source/manage/client-proxysql-metrics.rst:44]: Then should this not be merged into 2.6.0? This is a conflict between it and master.

**Pull request closed without merging**